examples/example_tissuemask.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>please name the variable as <strong>wsi </strong>instead of wsi_reader</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>